README.md

```diff
@@ -1,224 +1,147 @@
+## Problem Statement
 
-Almost all users of the modern web expect a high level of responsiveness and a
-_seamless_ experience when visiting a website. There is research that shows that
-40 percent of visitors to a website will leave if the site takes more than 3
-seconds to load. This number becomes higher when mobile visitors are taken into
-account.
+When it comes to making engaging web sites, we often find ourselves needing to
+send a lot of data (text, images, media, etc.). But sending and loading this
+**huge** data payload when your first land on a page **feels** slow,
+especially on a slow connection.
 
-As web developers, making sure the websites we build are fast and responsive to
-users is critical. But when it comes to loading content, we often find ourselves
-needing a lot of data.
+However, web users expect sites to load quickly and to stay updated. Research
+shows that 40 percent of visitors to a website will leave if the site takes
+more than 3 seconds to load. Mobile users are even less patient.
 
-Data takes memory to store, and bandwidth to download from a server. But just as
-importantly to the end user, loading a lot of data at once can also take a
-noticeable amount time, especially on a slow connection.
+We deliver sites that don't bore users by using a technique called ***AJAX***.
+In AJAX we:
 
-JavaScript provides a solution by allowing us to serve up data content
-_separately_ from the initial page load of a site. The result is that a user
-visiting a website will see _something_ load quickly, the initial DOM. When the
-rest of the data is ready, JavaScript can be used to add it to the DOM.
+1. Deliver an initial, engaging page using HTML and CSS
+2. Use JavaScript to add more to the DOM, behind the scenes
 
-This opens up a lot of possibilities!
+AJAX relies on several technologies:
 
-* It saves on bandwidth. We're not loading the entire page over any over, only
-what we need
-* It allows us to pull in dynamic content. The same HTML page could be used for
-every recipe on a cooking website, only the text content changes.
-* It allows us to get data from multiple sources. We could make a website that
-displays the current weather forecast and the current price of bitcoin side by
-side!
-
-Overall, allowing JavaScript to request, receive, and render data results in a
-much better user experience. In this lesson, we will be discussing how
-JavaScript retrieves data along with how to use the built-in `fetch()` function to
-handle remote data retrieval.
-
-## Objectives
+* Things called `Promise`s
+* Things called `XMLHttpRequestObject`s
+* A "serialization format" called JSON
+* "asynchronous Input / Output"
+* "the event loop"
 
-1. Explain how JavaScript fetches data from remote resources
-2. Explain how `fetch()` is used in modern browsers
-3. Explain what are sending `fetch()` requests 
-4. Working around backwards compatibility issues
+Understanding each of these first gets you away from _experiencing_ the magic
```

Comment:
I'm not really understanding what "gets away from" means here. I'd rephrase a bit like this:
In order to understand the JavaScript `fetch()` technique, it's not really necessary to understand exactly how all of the about AJAX functionality works. We're going to gloss over all of these pieces in this lesson, and by doing so you'll have a solid understanding of how `fetch()` works.